src/test/resources/descriptor-with-quotas.yaml

```diff
@@ -0,0 +1,36 @@
+---
+context: "contextOrg"
+source: "source"
+projects:
+  - name: "foo"
+    consumers:
+      - principal: "User:App0"
+      - principal: "User:App1"
+    producers:
+      - principal: "User:App0"
+      - principal: "User:App2"
+    topics:
+      - name: "foo"
+        config:
+          replication.factor: "1"
+          num.partitions: "1"
+        consumers:
+          - principal: "User:App3"
+        producers:
+          - principal: "User:App4"
+      - name: "bar"
+        config:
+          replication.factor: "1"
+          num.partitions: "1"
+platform:
+  kafka:
+    quotas:
```

Comment:
Resonable question. I have open design questions here:

* Should this go at the level of producers and consumer below? 
* Should we create a template? something like defines a label and this would create you default quotas as configured?

Core objective, try to make this section more digestible ;-)  

What do you think?

Reply:
I think that it's a good idea.
At , the second point, a template would be a separate file? Can give me an example?

The principal idea it's to use a separate model of gitops used by differents squads in an organization, with the options to be changed.
To put it in context, a development squad will define a quota usage for his users/principals, and the infra team can define quotas for a specific intervention in a cluster due to problems for a zombi consumption/production principal